src/test/scala/chiselTests/util/SparseVecSpec.scala

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
I've been putting tests that involve Verilog simulation in the integration tests, I don't feel strongly that they need to be there but maybe worth moving these over.

Reply:
Good idea. Moved over in https://github.com/chipsalliance/chisel/pull/3619/commits/fbd64e040becffed8cfaf404537b3df51c02fb1c.